README.md

```diff
@@ -42,19 +43,38 @@ health-checker [options]
 
 | Option | Description | Default 
 | ------ | ----------- | -------
-| `--port` | The port number on which a TCP connection will be attempted. Specify one or more times. | | 
+| `--port` | The port number on which a TCP connection will be attempted. Can be specified multiple times. | |
+| `--http` | The url:port to check for a 2XX status code. Can be specified multiple times. | |
+| `--script` | Path to an executable script that should return with a 0 exit status if successful. Can be specified multiple times. | |
```

Comment:
I can't remember if health-checker supports this, but if not, we should file an issue to track it and come back to it later: timeouts. We should be sure to support (configurable) timeouts for making TCP connections, HTTP calls, and running scripts, guaranteeing that health-checker always responds within some SLA whether or not the underlying pieces work.

Reply:
https://github.com/gruntwork-io/health-checker/issues/10